src/parse.py

```diff
@@ -49,6 +50,17 @@ def open_json_file(filename):
         raise ValueError('{} is not valid JSON.'.format(filename))
 
 
+def validate_object_keys(keys: Union[Tuple[str], List[str]], test_json):
+    if len(keys):
```

Comment:
where is the test for empty keys?

Reply:
test_if_empty_keys_raise_exception_with_empty_key added